lesson15/js/main.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+const openModalBtn = document.getElementById('js-openModalBtn');
+openModalBtn.addEventListener(
+	'click',
+	() => {
+		const getModal = document.getElementById('js-modalContainer');
+		getModal.classList.add('is-visible');
+		const modalBg = document.createElement('div');
+		modalBg.id = 'js-bgFixed';
+		modalBg.classList.add('el_modalBg');
+		document.body.appendChild(modalBg);
+		document.body.style.position = 'fixed';
+		closeButton();
+	},
+	false
+);
+
+function closeModal() {
+	const getModal = document.getElementById('js-modalContainer');
+	getModal.classList.remove('is-visible');
+	const bgFixed = document.getElementById('js-bgFixed');
+	document.body.removeAttribute('style');
+	bgFixed.remove();
+}
+
+function closeButton() {
+	const closeModlBtn = document.getElementById('js-closeModal');
+	closeModlBtn.addEventListener(
+		'click',
+		() => {
+			closeModal();
+		},
+		false
+	);
+}
+
+const form = document.getElementById('js-loadmenuBtn');
+form.addEventListener(
+	'click',
+	(event) => {
+		event.preventDefault();
+		const inputNumber = document.getElementById('number');
+		const inputText = document.getElementById('words');
+		if (!inputNumber.value === false && !inputText.value === false) {
```

Comment:
IMO
With the current conditional expression, even if you enter a blank field in the text input field (input[type="text"]), the request will be generated.
It's not in the specs of the assignment in that much detail, so I gave it an IMO.

Reply:
Thank you for review!
Certainly, it would be a problem if a blank was sent, so I fixed it so that a blank would also be an error.

[1f06d7a](https://github.com/nyattsu72/hanson-frontend-cource/pull/20/commits/1f06d7a893a112df037215aa8842fa75acb9b1f5